src/main/java/net/feed_the_beast/launcher/json/versions/Library.java

```diff
@@ -68,11 +68,11 @@ public String getPathNatives () {
         if (_artifact == null) {
             _artifact = new Artifact(name);
         }
-        return _artifact.getPath(natives.get(OS.CURRENT).replace("${arch}", (Settings.getSettings().getCurrentJava().is64bits ? "64" : "32")));
```

Comment:
I personally like to use extra parentheses with ?: after seeing bad code and bugs